userspace/libscap/scap_savefile.c

```diff
@@ -2716,7 +2716,11 @@ int32_t scap_next_offline(scap_t *handle, OUT scap_evt **pevent, OUT uint16_t *p
 		if(readsize != sizeof(bh))
 		{
 			int err_no = 0;
+#ifdef WIN32
+			const char* err_str = "read error";
+#else
 			const char* err_str = gzerror(f, &err_no);
+#endif

```

Comment:
How is this even compiled for Windows with the `!_WIN32` guard above? How does WIN32 compare to _WIN32?

Reply:
I am not sure which _WIN32 guard you refer to.

But I replaced the _WIN32 defines with WIN32 ones for consistency